insights/client/config.py

```diff
@@ -242,6 +242,10 @@
         # non-CLI
         'default': os.path.join(constants.default_conf_dir, 'remove.conf')
     },
+    'tags_file': {
+        # non-CLI
+        'default': os.path.join(constants.default_conf_dir, 'tags.conf')
```

Comment:
should be `tags.yaml`

Reply:
ah! didn't realize we had made that switch. On it